lib/config-migration.js

```diff
@@ -407,7 +411,11 @@ function migrateComplianceTriggers (config) {
       createTrigger(requirements.sanctions, global.sanctionsVerificationThreshold)
     )
   }
-
+  if (global.hardLimitVerificationActive && _.isNumber(global.hardLimitVerificationThreshold)) {
```

Comment:
Isn't checking for `global.hardLimitVerificationActive` redundant?

Reply:
I'm not sure about this, but I believe it's possible that on 7.4 admin the hardLimitVerificationActive is disabled (false), and there's a threshold somewhere in the config, for example, if the hardLimit was ever enabled.

Reply:
Ah sorry, I read the variable wrong. You're correct!